content/blog/2024-01-07-INTA/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Citizen science and governance models
+author: Yanina Bellini Saibene
+summary: " "
+date: '2024-01-07'
+categories:
+  - English
+  - Community
+  - 100DaysToOffload
+tags:
+  - English
+  - Community
+  - 100DaysToOffload
+---
+
+{{< figure src="featured.jpeg" alt="Field day at INTA. INTA's researcher showing soil trials and other related research activities and results">}}
+
+Continuing with the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.
+
+## INTA 
+
+INTA is an Argentine federal research and extension agency in charge of the generation, adaptation and diffusion of technologies, knowledge and learning procedures for the agriculture, forest and agro-industrial activities within an ecologically clean environment. It was created in 1956 and have 44 experimental agricultural stations and 240 extension units around the country.  It is the research institution with the largest territorial presence in our country. 
+
+In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.
```

Comment:
"After the return to democracy in 1983, in 1986 the president issued Decree 287"